volume/local/local_unix.go

```diff
@@ -157,6 +162,18 @@ func (v *localVolume) postMount() error {
 	return nil
 }
 
+func (v *localVolume) unmount() error {
+	if v.needsMount() {
+		if err := mount.Unmount(v.path); err != nil {
+			if mounted, mErr := mountinfo.Mounted(v.path); mounted || mErr != nil {
```

Comment:
I thought I kill all those redundant checks already... Anyway, it's a separate issue.